specs/logging_changes_spec.adoc

```diff
@@ -0,0 +1,171 @@
+
+Since a new logging module is being introduced in Tendrl-common(https://github.com/Tendrl/specifications/pull/94), after its implementation
+there will be requirement of changes in all the modules of Tendrl where logging is used.
+This file proposes all those changes and how they will be made.
+
+
+## Problem description
+
+Many modules in Tendrl have a functionality of logging different types of situations/use cases such as logging (info, errors, warning etc.).
+Till now these were implemented by importing the Python standard library's logging module then calling logger functions to handle the logging.
+After implementation of new logging module there will be requirement of changes in all the modules of Tendrl where logging is used.
+
+#### Use Cases
+
+There will be 3 classes(use cases) that will be provided by the common logging module:
+
+ * Log - Will write logs to log files.
+ * Alert - Will send the alert messages to the node agent socket and write the same to the log files.
+ * Operations - Will write operation log to log files.
+
+#### Proposed change
+
+Earlier there was only provision of writing to log files, now this functionality in being diversified and extended.
+
+The following are the key points to be noted:
+
+* Introduction of alert and operations along with logs and all these will be treated as messages emitted by Tendrl modules.
+
+* There will 3 classes of messages that will be provided and will be used to achieve the required purpose:
+
+ ... Log - Will write logs to log files.
+ ... Alert - Will send the alert messages to the node agent socket and write the same to the log files.
+ ... Operations - Will write operation log to log files.
+
+* *Importing logging and calling logger functions* will be replaced by importing the respective class from the common for the desired functionality
+and calling the message handler function provided with the same.
+
+*_Example change :_*
+
+----
+import logging
+
+LOG = logging.getLogger(__name__)
+
+LOG.info("%s complete" % self.__class__.__name__)
+----
+*Replaced by :*
+----
+from tendrl.commons.new_logging import Log, Alert, Operations
+
+message = Log(parameters)
+
+message.handler()
+----
+
+#### Alternatives
+
+None
+
+#### Data model impact:
+
+None
+
+#### Impacted Modules:
+
+#### Tendrl/common impact
+
+Similar changes like _example change_ mentioned above will be required in following modules :
+
+* /commons/etcdobj/etcdobj.py
+* /commons/flows/base_flow.py
+* /commons/manager/manager.py
+* /commons/manager/rpc_job_process.py
+* /commons/persistence/persister.py
+* /commons/utils/ansible_module_runner.py
+* /commons/utils/command.py
+* /commons/utils/package_installer.py
+* /commons/utils/service.py
+* /commons/config.py
+* /commons/log.py
+
+#### Tendrl/node_agent impact
+
+Similar changes like _example change_ mentioned above will be required in following modules :
+
+* /node_agent/manager/manager.py
```

Comment:
Listing individual sources not required I feel and you just mention to replace all the logging instances with new framework.